### PR TITLE
Added GoLang resources in field of AI

### PR DESCRIPTION
**Pull request by kelly644**

### CODE REVIEW

**Comment by kelly644** [comment]
As a beginner in open source and being my first PR , I would very much appreciate if its counted towards hacktoberfest PR

> an individual PR can be opted-in with a maintainer adding the "hacktoberfest-accepted" label to the PR

File: README.md [hunk 1]
```diff
@@ -60,6 +60,7 @@ Awesome Artificial Intelligence, Machine Learning and Deep Learning as we learn
         - [Reinforcement learning](./details/julia-python-and-r.md#reinforcement-learning)
       - [ML on Code/Programm/Source Code](./ML-on-code-programming-source-code.md)
     - [Programming in Python](Programming-in-Python.md)
+    - [AI in Golang](./details/julia-python-and-r.md#programming-in-golang)
```
Comment (neomatrix369) [inline README.md:63]: Happy to have a new category for GoLang!
And also backlink it from the main README page(s)

**Review by neomatrix369 — CHANGES_REQUESTED** [review]
LGTM 

Thanks @kelly644

File: README.md (hunk 1 above)
Comment (kelly644) [reply README.md:63]: I had already added them in this PR

Comment (neomatrix369) [reply README.md:63]: I meant Golang as a sub-topic on its own: 

### Golang
<your links>

**Comment by neomatrix369** [comment]
> As a beginner in open source and being my first PR , I would very much appreciate if its counted towards hacktoberfest PR
> 
> > an individual PR can be opted-in with a maintainer adding the "hacktoberfest-accepted" label to the PR

Thanks for your PR but today is the last day and as per the new rules the PR may not make it into event - I'm unsure about it

**Comment by neomatrix369** [comment]
If you can apply the changes I requested quick enough we can try to merge this PR

**Comment by neomatrix369** [comment]
I\ll merge this one but there is no guarantee if it is counted for HF or now. Good luck with it though.

**Pull request merged**